ansible/roles/kong-api/defaults/main.yml

```diff
@@ -4660,4 +4662,74 @@ kong_apis:
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
 
+  - name: createGroup
+    uris: "{{ group_service_prefix }}/v1/create"
+    upstream_url: "{{ group_service_url }}/v1/group/create"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'groupCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: updateGroup
+    uris: "{{ group_service_prefix }}/v1/update"
+    upstream_url: "{{ group_service_url }}/v1/group/update"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'groupUpdate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"      
 
+  - name: listGroup
+    uris: "{{ group_service_prefix }}/v1/list"
+    upstream_url: "{{ group_service_url }}/v1/group/list"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'groupList'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"      
+
+  - name: readGroup
+    uris: "{{ group_service_prefix }}/v1/read"
+    upstream_url: "{{ group_service_url }}/v1/group/read"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'groupRead'
```

Comment:
This is incorrect ACL, Follow the correct ACL convention from the below link,
https://project-sunbird.atlassian.net/wiki/spaces/DevOps/pages/1202847861/Best+Practces+For+Onboarding+Consumers+In+Sunbird